pkgs/by-name/sh/shader-slang/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  python3,
+  llvmPackages_13,
+  xorg,
+}:
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes